common/api-review/app.api.md

```diff
@@ -96,6 +110,15 @@ export function initializeApp(options: FirebaseOptions, config?: FirebaseAppSett
 // @public
 export function initializeApp(): FirebaseApp;
 
+// @public
+export function initializeServerApp(options: FirebaseOptions | FirebaseApp, config: FirebaseServerAppSettings): FirebaseServerApp;
+
+// @internal (undocumented)
+export function _isFirebaseApp(obj: FirebaseApp | FirebaseOptions): obj is FirebaseApp;
+
+// @internal (undocumented)
+export function _isFirebaseServerApp(obj: FirebaseApp | FirebaseServerApp): obj is FirebaseServerApp;
```

Comment:
Perhaps they should be in utils then?

Reply:
Talked to @hsubox76 about this, and the guidance is not to put it into the `util` package. `util` currently doesn't depend an `app`, but moving this method to that `util` would require us to import `app` in order to access the `FirebaseServerApp` type declaration.

Importing `app` from `util` could set us up for a circular dependency in the future, so it would be better if this method just stayed in `app` itself.

Reply:
Decision: will keep these methods public.